code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -120,7 +120,7 @@
 	src.dump_contents()
 
 	src.opened = 1
-	playsound(src.loc, open_sound, 15, 1, -3)
+	playsound(src, 'sound/effects/locker_open.ogg', 40, 1, -3)
```

Comment:
For what purpose?

Reply:
shit fix for a percieved bug, it actually didn't do anything at all because the sound wasn't playing due to the overlap of 30 sounds at the same time shitting on byond